grpc/py-cubic/cubic/client.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright(2019) Cobalt Speech and Language Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http: // www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import grpc
+
+from google.protobuf.empty_pb2 import Empty
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__))))
+from cubic_pb2_grpc import CubicStub
+from cubic_pb2 import RecognitionConfig, RecognitionAudio
+from cubic_pb2 import ListModelsRequest, RecognizeRequest, StreamingRecognizeRequest
+
+class Client(object):
+    """ A client for interacting with Cobalt's Cubic GRPC API."""
+
+    def __init__(self, serverAddress, secure=True,
+                serverCertificate=None,
+                clientCertificate=None,
+                clientKey=None,
+                bufferSize=8192):
+        """  Creates a new cubic Client object.
+
+        Args:
+            serviceAddress: host:port of where cubic server is running
+
+            secure: If set to true, channel with transport security is used.
+                    Otherwise, an insecure grpc channel is used. Default is True.
+
+            serverCertificate:  PEM certificate as byte string which is used as a  
+                                root certificate that can validate the certificate 
+                                presented by the server we are connecting to. Use this 
+                                when connecting to an instance of cubic server that is 
+                                using a self-signed certificate.
+
+            clientCertificate:  PEM certificate as bytes string presented by this Client when
+                                connecting to a server. Use this when setting up mutually 
+                                authenticated TLS. The clientKey must also be provided.
+
+            clientKey:  PEM key as byte string presented by this Client when
+                        connecting to a server. Use this when setting up mutually 
+                        authenticated TLS. The clientKey must also be provided.
```

Comment:
The clientKey must also be provided.

`clientKey` or `clientCertificate`?

Reply:
`clientCertificate`, will change